requirements.txt

```diff
@@ -31,10 +28,17 @@ mypy==0.782
 # Upstream url: https://pypi.python.org/pypi/pytest-mock
 pytest-mock==1.1
 
+# PEP 484
+# License: PSF
+# Upstream url: https://github.com/python/typing
+typing-extensions==3.7.4
+
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
-amundsen-common>=0.3.6,<1.0
+amundsen-common==0.5.3
```

Comment:
should we still make it a range?

Reply:
Hmm, sure, we could make it `>=0.5.3`. This will need to be bumped anytime someone wants to use an updated common object (changes in 0.5.3 meant I had to change this in my PR from 0.5.2!), but we maybe don't care about all minor changes. 

- [ ] >=